### PR TITLE
docs(QTM-31): Updates versioning guide on contributting.md and…

### PR DESCRIPTION
**Pull request by toncabral88**
… contributors on readme.md

## Description
https://jirasoftware.catho.com.br/browse/QTM-31

## Review guide
- [ ] README.md
- [ ] Contributting.md

### CODE REVIEW

**Review by matheusrpp — DISMISSED** [review]
## Review guide
- [x] README.md
- [x] Contributting.md

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -2,10 +2,15 @@
 
 You can contribute in many ways: openning or participating of a issue, reporting a bug, sending a [pull request](#submitting-a-pull-request).
 
+## Contribution Prerequisites
+
+- Node installed at v12.16.1 and Yarn.
```
Comment (rapahaeru) [inline CONTRIBUTING.MD:7]: here, we should put 
"Node installed at v12.16.1 and Yarn. (to make it easier, we have .nvmrc, so just use `nvm use` to sets the repo default version of node)"

What do you think?

File: CONTRIBUTING.MD [hunk 2]
```diff
@@ -25,17 +30,27 @@ You can contribute in many ways: openning or participating of a issue, reporting
 
     Note, using a `git add .` will automatically add all files. You can do a `git status` to see your changes, but do it before `git add`.
 
-6.  Commit your changes using a descriptive commit message.
-
+6.  Commit your changes using [commitizen](http://commitizen.github.io/cz-cli/).
     ```sh
-    git commit -m "Brief Description of Commit"
+    yarn commitizen
```
Comment (rapahaeru) [inline CONTRIBUTING.MD:35]: "yarn commitizen" (remember, before of to use commitizen, the packages should be installed previously)

WDYT?

File: CONTRIBUTING.MD [hunk 3]
```diff
@@ -45,4 +60,4 @@ Open your pull request, if you don't know how do it, check this [tutorial](https
 
 For the good manutance of the project, follow all the code styles:
 - [Editorconfig](http://gitlab.devel/frontend-platform/quantum/blob/master/.editorconfig)
-- [Eslint Airbnb](http://gitlab.devel/frontend-platform/quantum/blob/master/.eslintrc.js)
+- [Eslint Airbnb](http://gitlab.devel/frontend-platform/quantum/blob/master/.eslintrc.js)
```
Comment (rapahaeru) [inline CONTRIBUTING.MD:63]: i didnt understand this references

**Review by rapahaeru — CHANGES_REQUESTED** [review]
i think the `maintainers.md` file should be updated too.
add our names (and e-mail) in there.

File: CONTRIBUTING.MD [hunk 4]
```diff
@@ -25,17 +30,27 @@ You can contribute in many ways: openning or participating of a issue, reporting
...
+
+8.  Push your commits to your Fork:
 
     ```sh
     git push -u origin branch-name
     ```
-8.  Submit a pull request.
+9.  Submit a pull request.
 
 ## Submitting a Merge Request
```
Comment (rapahaeru) [inline CONTRIBUTING.MD:55]: Submitting a **Pull** Request

File: CONTRIBUTING.MD [hunk 5]
```diff
@@ -2,10 +2,15 @@
 
 You can contribute in many ways: openning or participating of a issue, reporting a bug, sending a [pull request](#submitting-a-pull-request).
 
+## Contribution Prerequisites
+
+- Node installed at v12.16.1 and Yarn.
+- Docker for the visual tests.
+
 ## Getting Started
 1.  Fork the project, [Help Guide to Fork a repository](https://docs.gitlab.com/ce/gitlab-basics/fork-project.html/).
```
Comment (rapahaeru) [inline CONTRIBUTING.MD:11]: updates the url to github

**Review by matheusrpp — APPROVED** [review]
## Review guide
- [x] README.md
- [x] Contributting.md

File: CONTRIBUTING.MD [hunk 6]
```diff
@@ -25,24 +33,33 @@ You can contribute in many ways: openning or participating of a issue, reporting
...
     ```sh
     git push -u origin branch-name
     ```
-8.  Submit a pull request.
+9.  Submit a pull request.
 
-## Submitting a Merge Request
+## Submitting a Pull Request
 
 Open your pull request, if you don't know how do it, check this [tutorial](https://docs.gitlab.com/ee/gitlab-basics/add-merge-request.html/)
```
Comment (rapahaeru) [inline CONTRIBUTING.MD:60]: we are using a gitlab url here. use a github one.
(i did updated in catho-components a same url)

File: CONTRIBUTING.MD [hunk 7]
```diff
@@ -25,24 +33,33 @@ You can contribute in many ways: openning or participating of a issue, reporting
...
 
 ## Code Style
 
 For the good manutance of the project, follow all the code styles:
-- [Editorconfig](http://gitlab.devel/frontend-platform/quantum/blob/master/.editorconfig)
-- [Eslint Airbnb](http://gitlab.devel/frontend-platform/quantum/blob/master/.eslintrc.js)
+- [Editorconfig](http://gitlab.devel/frontend-platform/quantum/blob/master/.editorconfig)
```
Comment (rapahaeru) [inline CONTRIBUTING.MD:65]: same here, this link does not exist.
I think is better to remove all this block

File: MAINTAINERS.md [hunk 8]
```diff
@@ -1,6 +1,5 @@
 ## Maintainers
 
-- Alan Oliveira ([alan.oliveira@catho.com](mailto:alan.oliveira@catho.com?Subject=Candidate%20Home%20Page%20Issue))
-- Allysson dos Santos ([allysson.santos@catho.com](mailto:allysson.santos@catho.com?Subject=Candidate%20Home%20Page%20Issue))
-- Daniel da Silva ([daniel.silva@catho.com](mailto:daniel.silva@catho.com?Subject=Candidate%20Home%20Page%20Issue))
-- Gabriel Gonçalves Daltoso ([gabriel.daltoso@catho.com](mailto:gabriel.daltoso@catho.com?Subject=Candidate%20Home%20Page%20Issue))
+- Raphael Oliveira ([raphael.oliveira@catho.com](mailto:raphael.oliveira@catho.com?Subject=Candidate%20Home%20Page%20Issue))
+- Ton Cabral do Nascimento ([washington.cabral@catho.com](mailto:washington.cabral@catho.com?Subject=Candidate%20Home%20Page%20Issue))
+- Matheus Rodrigues Pires ([matheus.pires@catho.com](mailto:matheus.pires@catho.com?Subject=Candidate%20Home%20Page%20Issue))
```
Comment (rapahaeru) [inline MAINTAINERS.md:5]: Clemente is missing.

**Review by rapahaeru — CHANGES_REQUESTED** [review]

**Review by rapahaeru — APPROVED** [review]

**Review by matheusrpp — APPROVED** [review]

**Pull request merged**